### DISCUSSION TITLE
Where to donate money?

### DISCUSSION THREAD

**Discussion by azzamsa** [Kozea/Radicale#1551] [Ideas]
Hi, 👋🏽 

Thank you so much for Radicle. Where to donate money? Is there something like a Github sponsor?

Thanks 🥞

**Comment by pbiering** [comment]
@Unrud / @liZe : anything planned?

**Reply by liZe** [reply]
> @Unrud / @liZe : anything planned?

Nothing planned for me, feel free to do what you want as far as I’m concerned!

**Reply by pbiering** [reply]
@liZe : are there any running expenses, e.g. for the domain "radicale.org" or the TLS certificate?

**Reply by liZe** [reply]
Just the domain, the certificate is free. I don’t need donations for the domain :).

**Comment by pbiering** [comment]
@Unrud : any comment from your side?

**Comment by pbiering** [comment]
@azzamsa : I have created a Wiki page now: https://github.com/Kozea/Radicale/wiki/Donations
Feel free to follow hints

**Reply by azzamsa** [reply]
Thank you! 🥞
